front/src/store/index.js

```diff
@@ -52,6 +53,31 @@ export default new Vuex.Store({
     },
     logout ({ commit }) {
       commit('CLEAR_USER_DATA');
+    },
+    recoveryPasswordSendEmail ({ commit }, payload) {
+      return new Promise((resolve, reject) => {
+        axios.post(`${serverPath}api/ForgotPassword`,
+          { Email: payload.email }).then(resp => {
+          resolve();
+        }).catch(resp => {
+          reject(resp.data);
+        });
+      });
+    },
+    recoveryPasswordSendPassword ({ commit, state }, passwordandtoken) {
+      return new Promise((resolve, reject) => {
```

Comment:
just 
```
return axios.post(`${serverPath}api/ForgotPassword/ResetPassword`,
          {
            Email: state.email,
            Password: passwordandtoken.password,
            Token: passwordandtoken.token
          }
        );
```

Reply:
why we do send in upperCase, as i know default settings of JSON parcer in .Net Core webAPI translate lowerCase from front into upper for server and via versa

it means you can just write 
```
recoveryPasswordSendPassword ({ commit, state },  { email, password, token }) { // or { email, password, token }
  return axios.post(`${serverPath}api/ForgotPassword/ResetPassword`,  { email, password, token });
}
```

Reply:
it seems in other place we call api/forgotPassword/resetPassword

so, for api path is valid also lower camelCase